scripts/scripts/combine_sentences.py

```
@@ -0,0 +1,72 @@
+    return open(output_path, "w", encoding="utf-8")
+
+
+def get_content(input_path: str) -> str:
+    """Reads content from a file or stdin."""
+    if input_path == "-":
+        return sys.stdin.read().strip()
+    with open(input_path, "r", encoding="utf-8") as f:
+        return f.read().strip()
+
+
+def main(input: str, output: str):
```

Pylint (3.3.7)

[warning] Redefining built-in 'input'

(W0622)